src/components/map/MapService.js

```diff
@@ -843,13 +854,12 @@ goog.require('ga_urlutils_service');
          * layers.
          */
         this.getBackgroundLayers = function() {
-          var self = this;
-          if (!layers) {
-            return;
-          }
+          var that = this;
           return $.map(gaTopic.get().backgroundLayers, function(bodId) {
-            var retVal = {id: bodId,
-                          label: self.getLayerProperty(bodId, 'label')};
+            var retVal = {
+              id: bodId,
+              label: that.getLayerProperty(bodId, 'label')
+            };
```

Comment:
That might be true. Initially, background layers were named the same way in layermanager/catalog and in background selector; but that was not wished anymore. So we created separate translations for background layers. Not sure it's valid for every background layer we have...most probably yes.